### ISSUE TITLE
Allow Slider Limit to be disabled

### ISSUE THREAD

**Issue by telnetr4** [animate1978/MB-Lab#92]
MB-Lab was built originally to create _realistic_ humans. Sometimes, realism isn't what is wanted.

Currently all modifiers have a max of -1 to 1, however, with the removal of 4 lines in `def combine_morphings` this limit can be increased to -5 to 5. I am afraid that this might introduce new problems.

An addition of a GUI option to disable/enable this limit should be added. It most likely will require changing how/where this limit is added.

**Comment by animate1978** [comment]
I think it's a good idea, there have been complaints on Blenderartists before if I recall that some wish they could go past -1 / 1. Even I wish that we could.

My only concern is that the age value, Manuel made it clear he did not want to do underage characters, I can understand why, there are messed up people in this world and to have a character that is young, with the ability to go "nude" that this could pose an ethical issue. I am pretty sure those were his feelings and I can see where he comes from there. Not to say Blender artists are sick people at all, not my intention there, however if these potentially young models end up in someone's hands with less than honorable intentions.... it's a difficult thing to think about.

Other than THAT, I say go ahead and see what you can do.

**Comment by telnetr4** [comment]
> My only concern is that the age value

This slider change only applies to modifiers that can be found in Body Measures. Even if age value was affected, its highly unlikely that the figures would look under-age as this is an extrapolation.

**Comment by animate1978** [comment]
Ah ok then. I was not sure exactly which values you were talking about.

**Comment by telnetr4** [comment]
Update: figured out how to set the hard limits to different numerical values. Again, don't know if this could propagate problems.

**Comment by telnetr4** [comment]
The most basic, but probably least shareable/useful way, to impliment this would be by adding a check box to the preference page plus two textboxes to allow for the user to change the range.

**Comment by mklemarczyk** [comment]
I will prepare PR to change limits to soft limits with message that we do not guarantee proper working outside the default limits. @animate1978 is it acceptable? It is not worth to introduce a separate button for this use in my opinion. The UI can not be overgrown. Moreover, Blender introduce soft limits for that use.

**Comment by animate1978** [comment]
I think it's acceptable, I agree the GUI needs to stay compact, which is why there had been some GUI work done with the new additions.

**Comment by animate1978** [comment]
Upon further review this might be something to explore again...

**Comment by animate1978** [comment]
![Limits_removed_001](https://user-images.githubusercontent.com/479115/108526216-7a73c380-7296-11eb-9270-ebf5e4d2f885.png)


![Limits_removed_002](https://user-images.githubusercontent.com/479115/108526212-79db2d00-7296-11eb-9b99-0acaf14dc2c5.png)

The slider cannot by default change the values past -1.0 or 1.0 BUT you can type them into the GUI directly. This does however give some less than desirable results and in certain cases will cause the mesh to fold into itself or in the "Body" value will invert the character entirely.

More testing needs to be done before committing this to the repo

**Comment by animate1978** [comment]
Correction .... sliders don't go past 0 to 1 ....
